docs/nodeport.md

```diff
@@ -0,0 +1,24 @@
+## Kubernetes Node Port
```

Comment:
I suggest to include info specific for nginx service exposed via node port in the dex helm documentation.
It looks like whole content is specific to dex deployment only